src/main/java/book/store/service/order/impl/OrderServiceImpl.java

```diff
@@ -0,0 +1,114 @@
+package book.store.service.order.impl;
+
+import book.store.dto.order.OrderItemResponseDto;
+import book.store.dto.order.OrderRequestDto;
+import book.store.dto.order.OrderResponseDto;
+import book.store.dto.order.UpdateOrderStatusRequestDto;
+import book.store.exception.EntityNotFoundException;
+import book.store.mapper.OrderItemMapper;
+import book.store.mapper.OrderMapper;
+import book.store.model.CartItem;
+import book.store.model.Order;
+import book.store.model.OrderItem;
+import book.store.model.ShoppingCart;
+import book.store.repository.order.OrderItemRepository;
+import book.store.repository.order.OrderRepository;
+import book.store.repository.shopping.cart.ShoppingCartRepository;
+import book.store.service.order.OrderService;
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
```

Comment:
remove `@Transactional` under methods
```suggestion
@Service
@Transactional
```